engines/engine.cpp

```diff
@@ -631,8 +634,25 @@ void Engine::flipMute() {
 }
 
 Common::Error Engine::loadGameState(int slot) {
-	// Do nothing by default
-	return Common::kNoError;
+	Common::InSaveFile *saveFile = _saveFileMan->openForLoading(getSaveStateName(slot));
```

Comment:
It seems to me, the indentation here is two spaces instead of a tab.

Reply:
I'll double check the indentation